tests/improver-generate-topography-bands-mask/01-help.bats

```diff
@@ -11,7 +11,7 @@ usage: improver-generate-topography-bands-mask [-h]
                                                INPUT_FILE_STANDARD_OROGRAPHY
                                                OUTPUT_FILE
 
-Read input orography and landmask fields. Return a a cube of masks, where each
+Reads input orography and landmask fields. Creates a set of masks, where each
```

Comment:
As with @MarkWorsfold's previous comment - if changing "set" above, this could be changed to "series" or "collection".